src/main/java/arb/ui/ProjectCard.java

```diff
@@ -52,7 +52,8 @@ public ProjectCard(Project project, int displayedIndex) {
             contentsPane.getChildren().remove(deadline);
         }
 
-        status.setText("Status: " + project.getStatus());
+        status.setText(project.isOverdue() ? "OVERDUE" : project.getStatus().toString());
```

Comment:
I think the UI for status should still start with "Status: " e.g. `Status: OVERDUE`, `Status: DONE`, just to make it clear on the UI end.